.github/workflows/test.yml

```diff
@@ -23,26 +23,26 @@ jobs:
           git config --global user.email 'bot@opentermsarchive.org'
           git config --global core.autocrlf false
       - uses: actions/checkout@v2
+      - name: Start MongoDB (UNIX)
+        if: runner.os != 'Windows'
+        run: |
+          mkdir /tmp/test-database
+          mongod --dbpath /tmp/test-database --fork --logpath /tmp/mongodb-log
+      - name: Start MongoDB (Windows)
+        if: runner.os == 'Windows'
+        run: |
+          sc.exe config MongoDB start= auto
```

Comment:
Maybe add a comment pointing to https://github.com/actions/runner-images/issues/5949#issuecomment-1223837948 as this is really difficult to understand without the whole context

Reply:
Or I guess the git blame would be sufficient. Never mind.